content/operate/kubernetes/re-clusters/expand-pvc.md

```diff
@@ -0,0 +1,96 @@
+---
+Title: Expand PersistentVolumeClaim (PVC)
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: 
+linkTitle: Expand PVC
+weight: 15
+---
+
+PersistentVolumeClaims (PVC) are created by the Redis Enterprise for Kubernetes operator and use dy the RedisEnterpriseCluster (REC). PVCs are created with a specific size and can be expanded if the underlying storage class supports it.
```

Comment:
```suggestion
PersistentVolumeClaims (PVC) are created by the Redis Enterprise for Kubernetes operator and use the RedisEnterpriseCluster (REC). PVCs are created with a specific size and can be expanded if the underlying storage class supports it.
```

Reply:
Probably "used by" .

Reply:
Why are we referring to the 'operator and not 'you'?

Reply:
The operator is part of our Kubernetes software.